### ISSUE TITLE
Cannot load modules

### ISSUE THREAD

**Issue by shotikot** [denoland/deno#6022]
Every time i try to load modules like bcrypt, mongo and etc. code crashes and shows me this error.

`error: The system cannot find the path specified. (os error 3)`.

```
import { Application, Router } from "https://deno.land/x/oak/mod.ts";
import { init, mongoClient } from "https://deno.land/x/mongo/mod.ts";
```
here first line is executed correctly, but the second line throws an error.

**Comment by Pangoraw** [comment]
Maybe related to https://github.com/denoland/deno/issues/5982 if you are importing `ts` from a `js` file. Try downgrading to 1.0.2 if you are on 1.0.3

**Comment by shotikot** [comment]
So i can't use typescript modules with javascript?

**Comment by bartlomieju** [comment]
@shotikot this is a regression in `1.0.3` which will be fixed by #6000

**Comment by lucacasonato** [comment]
This should be fixed in 1.0.4

**Comment by shotikot** [comment]
I updated to 1.0.5 but still get the same error.
